client/ayon_core/tools/publisher/control.py

```diff
@@ -1453,6 +1613,9 @@ def _set_publish_has_crashed(self, value):
     def _get_publish_has_validation_errors(self):
         return self._publish_has_validation_errors
 
+    def _get_publish_has_validation_blocking_errors(self):
+        return self._publish_has_validation_blocking_errors
+
```

Comment:
I might be wrong, but I feel this should totally replace `_get_publish_has_validation_errors`.
In `stop_publish()`, we are not yet using blocking errors, but all errors, blocking or not.
What happens if you have only non-blocking errors during publish ? As they will appear in the error list, thus blocking the publish.

This feature is a change of behavior, not an additional one.
Before we were stuck with all errors. Now we stop on blocking errors but continue on non-blocking ones.

It makes sense to stop at any error type with the UI, but in batch mode, it must fail only if there is blocking errors, bypassing non-blocking ones.

Reply:
An `_is_interactive` method has been added to distinguish whether the code is running in interactive or batch mode, in order to adapt the behavior and to stop the publish process for non-blocking errors in interactive mode only.